# 
# Submitted-By: Robert Schwebel, 2006-02-11
# Committed-By: Robert Schwebel
#
# Error: 
#
# Does not compile with gcc-4. 
#
# Description: 
#
# - 
#
# State:
# 
# - not sent to upstream yet
#

diff -urN tcpdump-3.9.1/ipproto.h tcpdump-3.9.1-ptx/ipproto.h
--- tcpdump-3.9.1/ipproto.h	2004-04-29 00:02:23.000000000 +0200
+++ tcpdump-3.9.1-ptx/ipproto.h	2006-02-11 14:50:32.000000000 +0100
@@ -37,6 +37,8 @@
  * $FreeBSD: src/sys/netinet/in.h,v 1.38.2.3 1999/08/29 16:29:34 peter Exp $
  */
 
+#include "netdissect.h"
+
 extern struct tok ipproto_values[];
 
 #ifndef IPPROTO_IP
